bugtool/cmd/configuration.go

```diff
@@ -39,6 +41,27 @@ func setupDefaultConfig(path string, k8sPods []string, confDir, cmdDir string) (
 	return &c, save(&c, path)
 }
 
+func bpffsMountpoint() []string {
```

Comment:
We already have a separate module for listing mounts from `/proc/self/mountinfo` (which contains more details than `/proc/mounts`):

https://github.com/cilium/cilium/blob/master/pkg/mountinfo/mountinfo.go#L125-L135

The example of using it, also for checking BPFFS mount, is here:

https://github.com/cilium/cilium/blob/master/pkg/bpf/bpffs_linux.go#L69-L81

Reply:
Thanks @mrostecki, I'll take a look at that package now